rpcs3/Emu/RSX/RSXTexture.cpp

```diff
@@ -351,7 +351,9 @@ namespace rsx
 	u16 vertex_texture::get_exact_mipmap_count() const
 	{
 		u16 max_mipmap_count = static_cast<u16>(floor(log2(std::max(width(), height()))) + 1);
-		return std::min(mipmap(), max_mipmap_count);
+		max_mipmap_count = std::min(mipmap(), max_mipmap_count);
+
+		return !!max_mipmap_count ? max_mipmap_count : 1;
```

Comment:
!! should compile to a cmp 0 anyway afaik, same as > 0 or max(x, 1), it comes down to preference I think. This code will be cleaned up before merge; note that the zero likely comes from the tex.mipmap() call and not the preceding statement.